src/internal/const.js

```diff
@@ -11,6 +11,6 @@ export const ordinal = ['first', 'second', 'third', 'fourth', 'fifth'];
 
 export const namespace = 'fluture';
 export const name = 'Future';
-export const version = 2;
+export const version = 3;
```

Comment:
@davidchambers Note how I'm incrementing the version of the Fluture `@@type`. This is because in changing the internal structure of the `Sequence` instances, I've broken backwards compatibility. `interpret` expects `_actions` to be a `List` now, and not an `Array`.

Despite this *technically* being a "breaking change", I will not release it as a major version, but use a minor version instead. Detecting incompatible versions is really just there to improve error messages.